<ui version="4.0" >
 <class>MultiEditDialog</class>
 <widget class="QDialog" name="MultiEditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Multiline Editor</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>247</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Text</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTextEdit" name="textEdit" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>380</width>
        <height>247</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Blob</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Select File for BLOB:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLineEdit" name="blobFileEdit" />
         </item>
         <item>
          <widget class="QPushButton" name="blobFileButton" >
           <property name="text" >
            <string>&amp;Open...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item rowspan="3" row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item rowspan="3" row="2" column="1" >
        <widget class="BlobPreviewWidget" native="1" name="blobPreviewLabel" >
         <property name="minimumSize" >
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>150</height>
          </size>
         </property>
         <property name="wordWrap" stdset="0" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>74</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2" >
        <widget class="QPushButton" name="blobSaveButton" >
         <property name="toolTip" >
          <string>Save content of the current BLOB into chosen file</string>
         </property>
         <property name="text" >
          <string>&amp;Save...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>840</width>
        <height>420</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>&amp;Date to String</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>&amp;Date:</string>
         </property>
         <property name="buddy" >
          <cstring>dateTimeEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QDateTimeEdit" name="dateTimeEdit" >
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&amp;String Format:</string>
         </property>
         <property name="buddy" >
          <cstring>dateFormatEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="dateFormatEdit" />
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="nullCheckBox" >
     <property name="text" >
      <string>Insert &amp;NULL</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlobPreviewWidget</class>
   <extends>QWidget</extends>
   <header>blobpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>blobFileButton</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>dateFormatEdit</tabstop>
  <tabstop>blobFileEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MultiEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MultiEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
